DigitalLearningSolutions.Data/Models/User/CentreUserDetails.cs

```diff
@@ -1,18 +1,40 @@
-﻿namespace DigitalLearningSolutions.Data.Models.User
```

Comment:
Are the line endings in this file correct?

Reply:
It was LF, formatter made it CRLF when I was editing the file.